### ISSUE TITLE
Paragraph not indented properly in list with additional blank lines...

### ISSUE THREAD

**Issue by digitalmoksha** [markdown-it/markdown-it#287]
In a list, when paragraphs (or code blocks) are indented in the list, they become part of the list.  However, if there are multiple blank lines in the list item, then they are not included in the list.  Based on http://spec.commonmark.org/dingus/, they should be.  For example

```
- First item

   paragraph one

   paragraph two
```

and this 

```
- First item

   paragraph one


   paragraph two
```

should render the same, but they currently doesn't in markdown-it.

**Comment by puzrin** [comment]
Thanks for info. I think, this behaviour appeared in one of latest spec change, and we did not ported it yet or missed (if it was not covered in spec samples).

Let me know is this fix is urgent or can wait a couple of weeks.

**Comment by digitalmoksha** [comment]
You guys have been incredibly thorough, I'm guessing it's just a new addition that hasn't been ported yet.

No, it's not urgent, I just happened to run across it today.

**Comment by puzrin** [comment]
Solved in 8.0.0, CM spec updated 0.24 -> 0.26
